### PR TITLE
Fixing boiler heat exchanger initialize function

### PR DESCRIPTION
**Pull request by MAZamarripa**
## Fixes
- boiler heat exchanger initialize method

## Summary/Motivation:
fix bug with boiler heat exchanger initialization method

## Changes proposed in this PR:
- 
-

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

File: idaes/power_generation/unit_models/helm/splitter.py [hunk 1]
```diff
@@ -158,12 +158,12 @@ def sum_split(b, t):
         @self.Constraint(time, self.outlet_list, doc="Pressure constraint")
         def pressure_eqn(b, t, o):
             o_block = getattr(self, "{}_state".format(o))
-            return self.mixed_state[t].pressure == o_block[t].pressure
+            return self.mixed_state[t].pressure*1e-6 == o_block[t].pressure*1e-6
```
Comment (eslickj) [inline idaes/power_generation/unit_models/helm/splitter.py:161]: We should really use the scaling tools to do stuff like this, but it's okay for now.  Probably should revisit it at some point.

Comment (andrewlee94) [inline idaes/power_generation/unit_models/helm/splitter.py:161]: Not sure if this is standard for the Helmholtz models, but should we be hard coding these scaling factors, or using the scaling tools?

**Review by eslickj — APPROVED** [review]

File: idaes/power_generation/unit_models/helm/splitter.py (hunk 1 above)
Comment (eslickj) [reply idaes/power_generation/unit_models/helm/splitter.py:161]: Actually now that I looked at the rest of the model, this is in the ```calculate_scale_factors``` method.  That means if someone uses that the equations will be double scaled, so we should probably not make these splitter scaling changes.  Let me know if the splitter scaling isn't working for whatever reason and I'll take a harder look at it.

**Review by eslickj — CHANGES_REQUESTED** [review]

**Review by eslickj — COMMENTED** [review]
@MAZamarripa if you don't mind, since I need to combine this with the gas turbine, I can make the boiler heat exchanger changes in the gas turbine PR and the splitter changes shouldn't be needed. Would that be ok?  It would make it a bit easier for me to deal with, since the gas turbine PR is also waiting.

**Comment by andrewlee94** [comment]
Is this PR still needed, or has it been merged with #323?

**Comment by eslickj** [comment]
@andrewlee94, I did merge it with the gas turbine one, except for the hard coded scaling.  Unless @MAZamarripa, thinks otherwise, I think we can close it.

**Comment by andrewlee94** [comment]
I am told most of this has been merged into #323. Anything that remains can be done in another PR, so I will close this.

**Pull request closed without merging**